src/FakeItEasy/Configuration/PropertySetterConfiguration.cs

```diff
@@ -101,10 +78,8 @@ public IPropertySetterConfiguration WhenArgumentsMatch(Func<ArgumentCollection,
         }
 
         private static IPropertySetterConfiguration AsPropertySetterConfiguration(
-            IVoidConfiguration voidArgumentValidationConfiguration)
-        {
-            return new ProperySetterAdapter(voidArgumentValidationConfiguration);
-        }
+            IVoidConfiguration voidArgumentValidationConfiguration) =>
+            new ProperySetterAdapter(voidArgumentValidationConfiguration);

```

Comment:
This looks a little weird, with the body aligned with the parameter... in fact, it's one of the reasons I don't like the Java-style braces, because it causes the same issue when the signature line overflows. Maybe it would be better to keep a block-style body here.